CHANGELOG.md

```diff
@@ -15,6 +15,10 @@
 * [#84](https://github.com/rubocop-hq/rubocop-minitest/pull/84): New cops `AssertKindOf` and `RefuteKindOf` check for use of `assert_kind_of`/`refute_kind_of` instead of `assert(foo.kind_of?(Class))`/`refute(foo.kind_of?(Class))`. ([@fatkodima][])
 * [#85](https://github.com/rubocop-hq/rubocop-minitest/pull/85): Add autocorrect to `Rails/AssertEmptyLiteral` cop. ([@fatkodima][])
 
+### Bug fixes
+
+* [#101](https://github.com/rubocop-hq/rubocop-minitest/issues/101): Fix parsing error for `Minitest/LiteralAsActualArgument` when using splat or when invoking without parentheses. ([@tsmmark][])
```

Comment:
Good catch! On the other hand, `Minitest/LiteralAsActualArgument` cop is an unreleased feature, so the changelog entry is not required.

Reply:
Right, of course. Updated